ansible/roles/test/tasks/lag_minlink.yml

```diff
@@ -12,8 +12,27 @@
         login: "{{switch_login[hwsku_map[peer_hwsku]]}}"
       connection: switch
     
-    - pause:
-        seconds: "{{ wait_down_time }}"
+    - name: Set delay
+      set_fact:
+        delay: 5
+
+    - name: Set retries
+      set_fact:
+        retries: "{{ (wait_down_time / delay | float) | round(0, 'ceil') }}"
```

Comment:
could you combine the set_fact steps into one?

Reply:
No, the variables depend on each other. In this case we would get an undefined var error